### DISCUSSION TITLE
Toggle Drawer on desktop size

### DISCUSSION THREAD

**Discussion by francescotau** [saadeghi/daisyui#1841] [Q&A] [answered]
Hi, How do I get the drawer to behave on desktop screen like on mobile screen? I would like to always have a Drawer to toggle (show/hide) on any screen. 

I appreciate any help, Thank you

**Comment by saadeghi** [comment, marked as answer]
The first example here: https://daisyui.com/components/drawer/#drawer  
It behaves the same on both mobile and desktop

Let me know if you have any questions.

**Reply by francescotau** [reply]
Aaaaah thanks! Completely missed that “drawer-mobile” class in my  code! Thank you
